modules/openapi-generator/src/test/resources/integrationtests/typescript/custom-path-params-expected/api/matrixParams.service.ts

```diff
@@ -132,8 +132,7 @@ export class MatrixParamsService {
         }
 
         let localVarPath = `/complexMatrixParamExploded${this.configuration.encodeParam({name: "matrixParamExploded", value: matrixParamExploded, in: "path", style: "matrix", explode: true, dataType: "ComplexParams", dataFormat: undefined})}`;
-        return this.httpClient.put<any>(`${this.configuration.basePath}${localVarPath}`,
-            null,
+        return this.httpClient.request<any>('put', `${this.configuration.basePath}${localVarPath}`,
```

Comment:
A side effect for put/post requests without body defined in the spec. Previously it would be passed as `null` to `HttpClient` - now it's just omitted.
I don't believe it introduces any runtime behavior change though